doc/source/conf.py

```diff
@@ -38,7 +35,7 @@
 
 # General information about the project.
 project = u'Bandit'
-copyright = u'2016, OpenStack Foundation'
+copyright = u'2018, PyCQA'
```

Comment:
Like Ian mentioned, not sure you can change this.

Reply:
@sigmavirus24  @ericwb , ack that makes sense. Not at PC right now, but will make an amend.

Reply:
It was mentioned we could add them, but not remove?

Reply:
https://wiki.openstack.org/wiki/Documentation/Copyright

Reply:
http://eavesdrop.openstack.org/meetings/security/2018/security.2018-04-26-15.00.log.html#l-45